test/doc-views/TR/TR.js

```diff
@@ -0,0 +1,216 @@
+exports.data = {
+    body: {
```

Comment:
Hi denis, the `missing class on body` issue we discussed yesterday, it turns out to because of the 'key'. `body` seems to be a remaining word. If I rename it to `body1` or `bodyEle`, the class would be in HTML as expected.

Reply:
Indeed, `body` is served as the placeholder in `express-handlebars`